scripts/ci/get-slack-user-id.sh

```diff
@@ -19,6 +19,7 @@ slack_user=''
 # TODO: please keep the list in the alphabetic order. It is simpler to maintain it this way.
```

Comment:
> please keep the list in the alphabetic order. It is simpler to maintain it this way.

phrased as a TODO, but doesn't hurt to follow already. Perhaps just move it to the "p"s below?